modules/users/server/config/strategies/jwt.js

```diff
@@ -0,0 +1,32 @@
+﻿'use strict';
+
+/**
+ * Module dependencies
+ */
+var passport = require('passport'),
+  JwtStrategy = require('passport-jwt').Strategy,
+  ExtractJwt = require('passport-jwt').ExtractJwt,
+  User = require('mongoose').model('User');
+
+module.exports = function (config) {
+  var opts = {
+    jwtFromRequest: ExtractJwt.versionOneCompatibility({ tokenQueryParameterName: 'auth_token' }),
```

Comment:
I needed to add to get our SocketIO to authenticate. `versionOneCompatibility()` will first look for the JWT Token in the header, if it doesn't find it there it will look for it in the querystring. Express will send the token in the header, and SocketIO will send it in the querystring.